apps/home/blocks.py

```diff
@@ -138,16 +138,17 @@ class TeaserBlockCentered(blocks.StructBlock):
         required=False
     )
     background_color = ColorChoiceBlock(
-        help_text=_('Not choosing a colour will result in a '
-                    'block with a white background.'),
+        help_text=_('Choose yellow, pink or purple to ensure CI conformity.'),
         required=False
```

Comment:
the help text reads a bit like this should be required?

Reply:
nice catch!